src/scheduler.h

```diff
@@ -39,7 +38,7 @@ class CScheduler
     CScheduler();
     ~CScheduler();
 
-    typedef boost::function<void(void)> Function;
+    typedef std::function<void(void)> Function;
```

Comment:
This change was implemented upstream as part of bitcoin/bitcoin#9605. Fine to remove here, any conflict should be simple to resolve.